src/extras/helpers/ArrowHelper.js

```diff
@@ -94,6 +92,9 @@ THREE.ArrowHelper.prototype.setLength = function ( length, headLength, headWidth
 	if ( headLength < length ){
 		this.line.scale.set( 1, length - headLength, 1 );
```

Comment:
Not sure what a zero-scaled line would look like - and we would still need to call updateMatrix anyway, in that case unconditionally.

I don't think doing this would bring any performance benefit, nor would it make the code easier to read.